source/loader/layers/sanitizer/linux/san_utils.cpp

```diff
@@ -84,4 +86,15 @@ void *GetMemFunctionPointer(const char *FuncName) {
     return ptr;
 }
 
+std::string DemangleName(const std::string &name) {
+    std::string result = name;
+    char *demangled =
+        abi::__cxa_demangle(name.c_str(), nullptr, nullptr, nullptr);
+    if (demangled) {
+        result = demangled;
+        free(demangled);
+    }
+    return result;
+}
```

Comment:
If "DemangleName" failed, the "KernelName" will be an empty string. 
How about "void DemangleName(std::string& name)"? We aren't interested in whether demangling succeeds or fails anyway.

Reply:
`std::string result = name;` makes sure if demangling failed, then the name is unchanged.

Reply:
Okay. Does "abi::__cxa_demangle" need to link a library?